Source/WebInspectorUI/UserInterface/Base/Main.js

```diff
@@ -2098,393 +2087,18 @@ WI._toggleInspectMode = function(event)
-                    // COMPATIBILITY (iOS 14.0): `Page.Setting.AdClickAttributionDebugModeEnabled` was renamed to `Page.Setting.PrivateClickMeasurementDebugModeEnabled`.
-                    {name: WI.UIString("Private Click Measurement Debug Mode"), setting: InspectorBackend.Enum.Page.Setting.PrivateClickMeasurementDebugModeEnabled, value: true},
-                    {name: WI.UIString("Ad Click Attribution Debug Mode"), setting: InspectorBackend.Enum.Page.Setting.AdClickAttributionDebugModeEnabled, value: true},
-                ],
-            ],
-        },
-        {
-            name: WI.UIString("%s:").format(WI.unlocalizedString("WebRTC")),
-            columns: [
-                [
-                    {name: WI.UIString("Allow Media Capture on Insecure Sites"), setting: InspectorBackend.Enum.Page.Setting.MediaCaptureRequiresSecureConnection, value: false},
-                    {name: WI.UIString("Disable ICE Candidate Restrictions"), setting: InspectorBackend.Enum.Page.Setting.ICECandidateFilteringEnabled, value: false},
-                    {name: WI.UIString("Use Mock Capture Devices"), setting: InspectorBackend.Enum.Page.Setting.MockCaptureDevicesEnabled, value: true},
-                    {name: WI.UIString("Disable Encryption"), setting: InspectorBackend.Enum.Page.Setting.WebRTCEncryptionEnabled, value: false},
-                ],
-            ],
-        },
-    ];
-
-    for (let group of settings) {
-        if (!group.columns.some((column) => column.some((item) => item.setting)))
-            continue;
-
-        let settingsGroupRow = table.appendChild(document.createElement("tr"));
-
-        let settingsGroupTitle = settingsGroupRow.appendChild(document.createElement("td"));
-        settingsGroupTitle.textContent = group.name;
-
-        let settingsGroupValue = settingsGroupRow.appendChild(document.createElement("td"));
-
-        let settingsGroupItemContainer = settingsGroupValue.appendChild(document.createElement("div"));
-        settingsGroupItemContainer.classList.add("container");
-
-        for (let column of group.columns) {
-            let columnElement = settingsGroupItemContainer.appendChild(document.createElement("div"));
-            columnElement.classList.add("column");
-
-            for (let item of column)
-                createCheckbox(columnElement, item.name, item.setting, item.value);
-        }
-    }
-
-    contentElement.appendChild(WI.ReferencePage.DeviceSettings.createLinkElement());
-
-    WI._deviceSettingsPopover.presentNewContentWithFrame(contentElement, calculateTargetFrame(), preferredEdges);
+    WI._deviceSettingsPopover = new WI.OverrideDeviceSettingsPopover(WI);
+    WI._deviceSettingsPopover.show(WI._deviceSettingsTabBarButton.element);
 };
 
+WI._handleDeviceSettingChanged = function()
+{
+    WI._deviceSettingsTabBarButton.activated = WI.deviceSettingsManager.defaultSettingsChanged;
```

Comment:
`defaultSettingsChanged` feels a bit ambiguous to me in this context. My initial reaction was that it was an event name because of how it is phrased. Maybe `hasOverridenDefaultSetting`?

Reply:
Done. Named `hasOverridenDefaultSettings`